test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -6067,10 +6084,9 @@ public async Task PackCommand_MultiTfm_PrereleaseDependency_WarningIsSuppressed_
 
                 // Act
                 msbuildFixture.RestoreProject(workingDirectory, projectName, string.Empty);
-                CommandRunnerResult result = msbuildFixture.PackProject(workingDirectory, projectName, $"/p:PackageOutputPath={workingDirectory}", validateSuccess: false);
+                CommandRunnerResult result = msbuildFixture.PackProject(workingDirectory, projectName, $"/p:PackageOutputPath={workingDirectory}", validateSuccess: true);
```

Comment:
Why was this false before?

Reply:
This test used to check for success itself but not log the output.  Tests in this class that expect success should let the `PackProject` method validate success since it will log a better message on failure.